docs/commands/super.md

```diff
@@ -966,33 +965,29 @@ FROM 'gha.bsup'
 ### The Test Results
 
 The following table summarizes the query performance for each tool as recorded in the
-[most recent archived run](https://super-cmd-perf.s3.us-east-2.amazonaws.com/2024-12-03_00-43-29.tgz).
+[most recent archived run](https://super-cmd-perf.s3.us-east-2.amazonaws.com/2024-12-27_21-58-22.tgz).
 The run time for each query in seconds is shown along with the speed-up factor
 in parentheses:
 
 |**Tool**|**Format**|**search**|**search+**|**count**|**agg**|**union**|
 |-|-|-|-|-|-|-|
-|`super`|`bsup`|6.4<br/>(2.0x)|14.3<br/>(1.4x)|5.8<br/>(0.03x)|5.7<br/>(0.03x)|8.2<br/>(64x)|
-|`super`|`parquet`|note 1|note 1|0.3<br/>(0.6x)|0.5<br/>(0.3x)|note 2|
-|`duckdb`|`db`|13.0<br/>(1x)|20.0<br/>(1x)|0.2<br/>(1x)|0.1<br/>(1x)|521<br/>(1x)|
-|`duckdb`|`parquet`|13.4<br/>(1.0x)|21.4<br/>(0.9x)|0.4<br/>(0.4x)|0.3<br/>(0.4x)|504<br/>(1.0x)|
-|`datafusion`|`parquet`|11.0<br/>(1.2x)|21.7<br/>(0.9x)|0.4<br/>(0.5x)|0.4<br/>(0.4x)|24.6<br/>(21x)|
-|`clickhouse`|`parquet`|71<br/>(0.2x)|870<br/>(0.02x)|1.0<br/>(0.2x)|0.9<br/>(0.2x)|72<br/>(7x)|
-|`clickhouse`|`db`|0.9<br/>(14x)|13.2<br/>(1.5x)|0.1<br/>(2.2x)|0.1<br/>(1.1x)|note 3|
-
-_Note 1: the `super` vectorized runtime does not yet support `grep`_
-
-_Note 2: the `super` vectorized runtime does not yet support array expressions_
-
-_Note 3: we were not able to successfully run the [union query](#union) with
+|`super`|`bsup`|6.4<br/>(1.9x)|12.5<br/>(1.6x)|5.8<br/>(0.03x)|5.6<br/>(0.03x)|8.2<br/>(64x)|
+|`super`|`parquet`|40.8<br/>(0.3x)|55.1<br/>(0.4x)|0.3<br/>(0.6x)|0.5<br/>(0.3x)|40<br/>(13.2x)|
+|`duckdb`|`db`|12.1<br/>(1x)|19.8<br/>(1x)|0.2<br/>(1x)|0.1<br/>(1x)|527<br/>(1x)|
+|`duckdb`|`parquet`|13.3<br/>(0.9x)|21.3<br/>(0.9x)|0.4<br/>(0.4x)|0.3<br/>(0.4x)|488<br/>(1.1x)|
+|`datafusion`|`parquet`|11.0<br/>(1.1x)|21.2<br/>(0.9x)|0.4<br/>(0.5x)|0.4<br/>(0.4x)|24.2<br/>(22x)|
+|`clickhouse`|`parquet`|70<br/>(0.2x)|829<br/>(0.02x)|1.0<br/>(0.2x)|0.9<br/>(0.2x)|71.4<br/>(7x)|
+|`clickhouse`|`db`|0.9<br/>(14x)|12.8<br/>(1.6x)|0.1<br/>(2.2x)|0.1<br/>(1.2x)|note|
+
+_Note: we were not able to successfully run the [union query](#union) with
 ClickHouse's beta JSON type_
 
 Since DuckDB with its native format could successfully run all queries with
 decent performance, we used it as the baseline for all of the speed-up factors.
 
 To summarize,
 `super` with Super Binary is substantially faster than multiple relational systems for
-the search use cases and performs on par with the others for traditional OLAP queries,
+the search use cases, and with Parquet performs on par with the others for traditional OLAP queries,
```

Comment:
Especially now that we have across-the-boards Parquet results, I figured I might as well clarify that those are the ones where `super` perf comes closest with OLAP queries. Looking forward to seeing CSUP showing the best performance. 🤞